### PR TITLE
Add issue/PR templates + root-level community guidelines

### LINKED ISSUE #559: Improve Community Guidelines
Currently, Community Guidelines are in:
- https://ivadomed.org/en/latest/contributing.html
- the repository wiki

It would be nice to also
- have a file, eg `CODE_OF_CONDUCT.md` --> only point to the available resources
- have issue and Pull-Request templates pointing to the relevant parts of the wiki

### PR DESCRIPTION
**Pull request by joshuacwnewton**
#### GitHub

- [X] I've given this PR a concise, self-descriptive, and meaningful title
- [X] I've linked relevant issues in the PR body
- [X] I've applied [the relevant labels](https://www.neuro.polymtl.ca/software/contributing#pr_labels) to this PR
- [X] I've assigned a reviewer

#### PR contents

- [X] I've consulted [ivadomed's internal developer documentation](https://github.com/ivadomed/ivadomed/wiki) to ensure my contribution is in line with any relevant design decisions
- [ ] ~~I've added [relevant tests](https://github.com/ivadomed/ivadomed/wiki/tests) for my contribution~~ **N/A**
- [ ] ~~I've updated the [relevant documentation](https://github.com/ivadomed/ivadomed/wiki/documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages~~ **N/A**

## Description

This PR:

* Adds 3 templates (1 PR and 2 issue), largely adapted from SCT/shimming-toolbox
* Adds a `CODE_OF_CONDUCT.md` that uses the [Contributor Covenant](https://en.wikipedia.org/wiki/Contributor_Covenant)
* Moves the `contributing.rst` to the project root to make it more visible, and adds a symlink within the docs, so that it can appear in both places at once.

## Linked issues

Fixes #559.

### CODE REVIEW

**Review by charleygros — APPROVED** [review]
That's exactly what we needed! Many thanks @joshuacwnewton

**Review by jcohenadad — APPROVED** [review]
Thank you @joshuacwnewton 🙏

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (joshuacwnewton) [inline CODE_OF_CONDUCT.md:64]: I've just noticed that the `CODE_OF_CONDUCT.md` has this blank for filling in contact information. I missed this on my first read-through. My apologies! :sweat_smile: 

@jcohenadad, does NeuroPoly have e a contact method that would be appropriate for private/sensitive communication such as this?

Comment (jcohenadad) [reply CODE_OF_CONDUCT.md:64]: hum, how about `jcohen [at] polymtl.ca` ?

**Pull request merged**